src/add-field-dropdown/add-field-dropdown.component.html

```diff
@@ -1,7 +1,5 @@
 <div *ngIf="!hidden" class="btn-group add-field-dropdown-container" dropdown keyboardNav="true" [isDisabled]="isDisabled" (onShown)="onDropdownShown()">
-  <button type="button" class="btn btn-add-field-dropdown" dropdownToggle>
-    <ng-content></ng-content> <i class="fa fa-caret-down"></i>
-  </button>
+  <i class="fa fa-plus fa-lg add-field-button" aria-hidden="true" dropdownToggle tooltip="Add Field" placement="right"></i>
```

Comment:
this will break an untested part, so maybe we can also introduce a test after fixing.

Reply:
Later I saw that, this change and other scss are breaking the UI of all other cases we use `add-field-dropdown`